build.gradle

```
@@ -2,14 +2,15 @@ plugins {
 	id 'org.springframework.boot' version '2.4.1'
 	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
 	id 'java'
+	id 'application'
 }
 
 group = 'uk.nhs.digital.nhsconnect'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
-bootJar {
-	mainClassName = 'uk.nhs.digital.nhsconnect.lab.results.IntegrationAdapterLabResultsApplication'
+application {
+	mainClass.set('uk.nhs.digital.nhsconnect.lab.results.IntegrationAdapterLabResultsApplication')
 }
```

We could simplify this further
```suggestion
mainClassName = 'uk.nhs.digital.nhsconnect.lab.results.IntegrationAdapterLabResultsApplication'
```
unless you think it's helpful to explicitly declare where the functionality is coming from.